### DISCUSSION TITLE
Customize the Module Bar by roles

### DISCUSSION THREAD

**Discussion by abdonrd** [directus/directus#11217] [Archive]
Right now the `Module Bar` config is on the `Project Settings`:
<img width="849" alt="Screenshot 2022-01-23 at 19 19 38" src="https://user-images.githubusercontent.com/1007051/150692313-2705aa6f-375d-4429-9edf-93d706eb1f80.png">

I would like to propose the option to customize it by roles.

**Comment by joselcvarela** [comment]
This is a duplicate of:
https://github.com/directus/directus/discussions/9076

Go there and upvote if you are interested.

Also check why we have done this way here:
https://github.com/directus/directus/pull/8012

**Reply by abdonrd** [reply]
I hadn't found that. Thanks for the reference!
